config.js

```diff
@@ -8,6 +8,10 @@ const { getOptionName } = require('./options/getOptionName');
 const { getRankedUnranked } = require('./options/getRankedUnranked');
 const { getRankedUnrankedUnderscored } = require('./options/getRankedUnrankedUnderscored');
 
+// TODO fix circular dependency
```

Comment:
todo?

Reply:
currently there is an issue with circular dependencies (https://github.com/online-go/gtp2ogs/pull/332#issuecomment-654633308), it was a reminder to fix it before merging
i should delete this after this is fixed, or possibly now too

Reply:
This is fixed, guess i forgot to remove this? 😅

Reply:
yes i forgot it too actually :sweat_smile: :sweat_smile: :sweat_smile: